.github/workflows/mo.yml

```diff
@@ -37,12 +40,20 @@ jobs:
           # MO requirements
           pip install -r requirements.txt
           pip install -r requirements_dev.txt
+          # requrements for CMake
+          sudo apt --assume-yes install libusb-1.0-0-dev
```

Comment:
Why the "libusb" is needed for CMake? I think it is needed for VPU plugin?

Reply:
@azhogov , Could you please comment this?

Reply:
it failed without it, just check

Reply:
pre-commit is failed w/o this line